x-pack/plugins/security_solution/server/lib/entity_analytics/risk_score/tasks/risk_scoring_task.ts

```diff
@@ -261,6 +281,14 @@ export const runTask = async ({
           range,
           runtimeMappings,
           weights: [],
+          alertSampleSizePerShard,
+        });
+        const tookMs = Date.now() - now;
+
+        runs.push({
```

Comment:
I decided not to add this to telemetry but to log it, maybe we do want individual run info in telemetry?